Dockerfile

```diff
@@ -1,15 +1,19 @@
 #Dockerfile for a Postfix email relay service
 FROM centos:latest
-MAINTAINER Juan Luis Baptiste juan.baptiste@gmail.com
+LABEL maintainer="Juan Luis Baptiste juan.baptiste@gmail.com"
 
 RUN yum install -y epel-release && yum update -y && \
     yum install -y cyrus-sasl cyrus-sasl-plain cyrus-sasl-md5 mailx \ 
-    perl supervisor postfix
+    perl supervisor postfix rsyslog
 RUN sed -i -e "s/^nodaemon=false/nodaemon=true/" /etc/supervisord.conf
 RUN sed -i -e 's/inet_interfaces = localhost/inet_interfaces = all/g' /etc/postfix/main.cf
+
+COPY etc/supervisord.conf /etc/
```

Comment:
Addig the config files can be resumed in one line using wildcards: COPY *.conf /etc